src/toil/job.py

```diff
@@ -1158,6 +1158,17 @@ def pre_update_hook(self) -> None:
         self._job_version += 1
         logger.debug("New job version: %s", self)
 
+    def get_job_type(self) -> str:
```

Comment:
What does this function do? It needs a docstring. It definitely *doesn't* return the "type" of the job in the sense of a Python class type.